modules/ROOT/assets/attachments/bundled-admin.yaml

```
@@ -2572,12 +2756,25 @@ paths:
                       use_stdlib_json:
                         description: Bypass the jsoniter package and use Go's stdlib instead
                         type: boolean
+                        default: false
                       http2:
                         type: object
                         properties:
                           enabled:
                             description: Whether HTTP2 support is enabled
                             type: boolean
+                            default: false
+                      allow_dbconfig_env_vars:
+                        description: Can be set to false to skip environment variable expansion in database configs
+                        type: boolean
+                        default: true
+                      diagnostic_interface:
+                        description: |-
+                          Network interface to bind diagnotic API to.
```

Not keen on the dangling preposition, but fix the typo anyway.

```suggestion
                          Network interface to bind diagnostic API to.
```